src/sentry/integrations/github/integration.py

```diff
@@ -50,9 +50,6 @@ def get_client(self):
     def get_repositories(self):
         return self.get_client().get_repositories()
 
-    def make_external_key(self, data):
```

Comment:
This was leftover and not needed since it's been added to the `GitHubIssueBasic` mixin.